content/docs/deployment/csmoperator/_index.md

```diff
@@ -14,29 +14,21 @@ Dell CSM Operator has been tested and qualified on Upstream Kubernetes and OpenS
 | -------------------------- | ------------------- |
 | 1.25, 1.26, 1.27           | 4.11, 4.12, 4.12 EUS |
 
-## Supported CSI Drivers
-
-| CSI Driver         | Version   | ConfigVersion  |
-| ------------------ | --------- | -------------- |
-| CSI PowerScale     | 2.5.0 +   |   v2.5.0 +     |
-| CSI PowerFlex      | 2.5.0 +   |   v2.5.0 +     |
-| CSI PowerStore     | 2.5.0 +   |   v2.5.0 +     |
-| CSI PowerMax       | 2.6.0 +   |   v2.6.0 +     |
-| CSI Unity XT       | 2.5.0 +   |   v2.5.0 +     |
+>NOTE:
+>- Authorization module installation with CSM operator with Openshift platform is not currently supported.
```

Comment:
Change line 18 to say  "Authorization module is only supported on Kubernetes platforms"